### ISSUE TITLE
[Bug]: Cannot add user if no tags are selected.

### ISSUE THREAD

**Issue by petras-sukys** [advplyr/audiobookshelf#1933]
### Describe the issue

Tried to add a very limited user (single library enabled, disabled all tags). Entered all the required info, pressed "Submit" and nothing happened. Tried various browsers - same thing. Developer console shows such error:

```
0123845.js:2 TypeError: Cannot read properties of undefined (reading 'length')
    at f.submitForm (ca51c9a.js:1:7532)
    at submit (ca51c9a.js:1:10602)
    at Ue (0123845.js:2:18718)
    at HTMLFormElement.n (0123845.js:2:11363)
    at c._wrapper (0123845.js:2:51088)
```

After exhausting all my ideas, created user the same, with all tags enabled, then edited the user to remove tags and got the popup message that at least one tag needs to be selected. Fair enough, would be great to have this error popup when adding (not only editing) user. Went back and tried to add user without any libraries added - error popup was shown.

### Steps to reproduce the issue

1. Go to <abshost>/config/users
2. Click on + to add new user
3. Enter username, password
4. Switch "Can Access All Tags" to OFF
5. Remove all tags
6. Try to save the user (nothing happens)

### Audiobookshelf version

2.3.2

### How are you running audiobookshelf?

Debian/PPA

**Comment by advplyr** [comment]
I fixed the bug there but currently it is required to add at least one tag. What was your goal with excluding all tags for the user?

**Comment by petras-sukys** [comment]
As I said, I just wanted to add very limited user, so switched all the toggles off, and then was not able to add the user.

I don't mind the requirement of at least one tag, I mind that there was no error message of why I'm not allowed to add a user :) Thanks for fixing that.

**Comment by advplyr** [comment]
Fixed in v2.3.3
